src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/Availability.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.model
+
+import java.time.LocalDate
+
+data class Availability(
+  val date: LocalDate,
+  val pendingBookings: Int,
+  val arrivedBookings: Int,
+  val nonArrivedBookings: Int,
+  val cancelledBookings: Int,
+  val lostBeds: Int
+)
```

Comment:
I think we just said to return the `availableBeds` here? I think this info might be interesting though, so maybe return the `availableBeds` in addition to the count?

Reply:
Oh wait, I'm getting ahed of myself here - so to be clear, this is a data class that has all the different types of bookings, which we then sum when calculating returning in the endpoint?

Reply:
```
Oh wait, I'm getting ahed of myself here - so to be clear, this is a data class that has all the different types of bookings, which we then sum when calculating returning in the endpoint?
```

that's right yeah, I suspect we'll need these counts elsewhere so figured the service layer should return them all then just transform it for the endpoint's response